ds248x/ds248x.go

```diff
@@ -15,6 +16,15 @@ import (
 	"periph.io/x/conn/v3/onewire"
 )
 
+// DS248xType for chip connected to the system identification (ds2482-100, ds2482-800, ds2483).
+type DS248xType uint8
+
+const (
+	isDS2482x100 = 0 // DS2482-100 selected
```

Comment:
You need to call out explicitly with the time. I recommend using iota, which will define the rest of the typed values for you.